x-pack/test/functional/apps/canvas/smoke_test.js

```diff
@@ -14,16 +14,15 @@ export default function canvasSmokeTest({ getService, getPageObjects }) {
   const retry = getService('retry');
   const PageObjects = getPageObjects(['common']);
 
-  describe('smoke test', async () => {
+  describe('smoke test', async function () {
+    this.tags('smoke');
     const workpadListSelector = 'canvasWorkpadLoaderTable canvasWorkpadLoaderWorkpad';
     const testWorkpadId = 'workpad-1705f884-6224-47de-ba49-ca224fe6ec31';
 
     before(async () => {
       // init data
-      await Promise.all([
-        esArchiver.loadIfNeeded('logstash_functional'),
-        esArchiver.load('canvas/default'),
-      ]);
+      await esArchiver.loadIfNeeded('logstash_functional');
+      await esArchiver.load('canvas/default');
```

Comment:
This change is required to keep order: we face several failures when `canvas/default` was started right when `logstash_functional` was still in progress.